### PR TITLE
updates to Java SDK, with queries

### PR DESCRIPTION
**Pull request by genwhittTTD**
Fresh update from UID2 with these mods from the UID2 version: 
- Name change
- Remove phone number
- Remove sharing.
- Update glossary link re personal data.
- Add new term to glossary, commented out
- Comment out link to new glossary term since it's for mobile

### CODE REVIEW

File: docs/sdks/sdk-ref-java.md [hunk 1]
```diff
@@ -54,14 +49,46 @@ The binary is published on the Maven repository:
...
+
+You will need to provide the values necessary for the SDK to authenticate with the EUID service.
+
+| Parameter | Description                                                                                | 
+| :--- |:-------------------------------------------------------------------------------------------|
+| `baseUrl/uid2BaseUrl` | The endpoint for the EUID service. See [Environments](../getting-started/gs-environments). | 
+| `clientApiKey` | The API key. See [EUID Credentials](../getting-started/gs-credentials).                    | 
+| `base64SecretKey` | The client secret. See [EUID Credentials](../getting-started/gs-credentials).              |
+
+(**GWH__JN the below table is from the EUID version. The one above is copied from UID2. The legacy EUID version is so different that I'm including it to double-check what the EUID doc should say.**)
```
Comment (jon8787) [inline docs/sdks/sdk-ref-java.md:68]: UID2 docs is source of truth, looks correct to me

File: docs/sdks/sdk-ref-java.md [hunk 2]
```diff
@@ -54,14 +49,46 @@ The binary is published on the Maven repository:
...
 
 | Parameter | Description | Recommended Value |
 | :--- | :--- | :--- |
 | `endpoint` | The endpoint for the EUID service. | N/A |
 | `authKey` | The authentication token that belongs to the client. For access to EUID, see [Contact Info](../getting-started/gs-account-setup.md#contact-info). | N/A |
 
-## Interface 
+### Interface (GWH__JN section copied from UID2)
+
+(**GWH__JN not sure what we need out of this section and the next which is legacy for EUID. Will EUID folks use the`SharingClient` class? And that "decrypt method" piece is not in the UID2 doc for Java SDK. I left the "Usage for Sharers" section out, but not sure about these encryption and decryption examples, whether they should be in.**)
```
Comment (jon8787) [inline docs/sdks/sdk-ref-java.md:79]: `SharingClient` will be used by EUID folks when Sharing becomes supported

File: docs/sdks/sdk-ref-java.md (hunk 1 above)
Comment (genwhittTTD) [reply docs/sdks/sdk-ref-java.md:68]: Updated, thx.

File: docs/sdks/sdk-ref-java.md (hunk 2 above)
Comment (genwhittTTD) [reply docs/sdks/sdk-ref-java.md:79]: Updated, thx.

**Pull request merged**